app/Scripts/HealthUIController.gd

```diff
@@ -4,15 +4,22 @@ extends CanvasLayer
 # Declare member variables here. Examples:
 # var a = 2
 # var b = "text"
-onready var healthLabel: Label = get_node("HealthLabel")
-
+onready var healthLabel = $HealthLabel
+onready var staminaLaber = $StaminaLabel
 # Called when the node enters the scene tree for the first time.
 func _ready():
 	healthLabel.text = "Health: %s" % 100
-
+	staminaLaber.text = "Stamina: %s" % 100	
 # Called every frame. 'delta' is the elapsed time since the previous frame.
 #func _process(delta):
 #	pass
 
-func _on_Player_health_update(amount: int) -> void:
-	healthLabel.text = "Health: %d" % amount
+
+func _on_Player_health_update(new_value):
```

Comment:
not going to be super strict about this cause it's just a game jam but i think typing hints can help catch bugs and make code more readable